src/views/product-integrations-landing/index.tsx

```diff
@@ -26,6 +30,11 @@ export default function ProductIntegrationsLanding({
 				sidecarSlot={<></>}
 			>
 				<div className={s.mainArea}>
+					<BrandedHeaderCard
+						productSlug={product.slug}
+						heading={`${product.name} Integrations`}
+						description={`A curated collection of official, partner, and community ${product.name} Integrations.`}
+					/>
```

Comment:
Forgot to call this out - we're generating the heading and description based on the product. If we're expecting this to be less strictly templated for future products, we could consider moving to a more directly authorable interface. For now this felt sufficient 👍

Reply:
Ah, I had a comment in the Asana ticket that was probably burred in the long description.

> I think the title of this page will always be ${product.name} Integrations, but I think the description may be something we want to be configurable on a per-product basis.  We probably will want to add a key to the [integrations config](https://github.com/hashicorp/dev-portal/blob/product-integrations/src/data/vault.json#L169-L185) which will allow authors to specify the description they want here.

I already am in talks with the Vault PMs who would like to change the description here to something more bespoke, so maybe we adjust in this PR before merging to be configuration driven.

Reply:
ah cool, sorry I missed that 😬 This sounds great to me, I'll adjust now!

Reply:
No worries at all!